functions.py

```diff
@@ -17,8 +17,10 @@ def ugly_load_to_sftp():
                           cursorclass=pymysql.cursors.DictCursor)
     df = pd.read_sql_query("CALL get_new_leads()", con)
 
-    df['TypeName'] = "ANTHROPOLOGY OLICO NPH STi"
-    df['TypeDesc'] = "00008625"
+    
+    df['TypeCode'] = "00008625"
+	df['TypeDesc'] = "ANTHROPOLOGY OLICO NPH STi                "
```

Comment:
Twee goed: Hoekom is daar so baie spaces na `STi`? En ook, jou indent op line 22 gaan 'n bug gooi dink (rerig DINK, nie seker nie).